src/main/scala/com/lightbend/conductr/sbt/LagomConductRPlugin.scala

```diff
@@ -0,0 +1,41 @@
+package com.lightbend.conductr.sbt
```

Comment:
This is a new plugin that is enabled when Lagom and the ConductR plugins are enabled. The plugin writes out Cassandra installation script contents.